cluster/manifests/nvidia/nvidia-driver-installer.yaml

```diff
@@ -22,6 +22,9 @@ spec:
         application: nvidia-driver-installer
         version: c117d39
     spec:
+{{ if index .ConfigItems "enable_rbac"}}
+      serviceAccountName: nvidia
```

Comment:
Should this have an else?

Reply:
Yes, but this is wrong in multiple places so we can fix it in a separate PR.